addons/crm_iap_lead_enrich/models/crm_lead.py

```diff
@@ -92,6 +92,7 @@ def iap_enrich(self, from_cron=False):
                                     {
                                         'type': 'simple_notification',
                                         'message': self.env.ref('crm_iap_lead_enrich.mail_message_lead_enrich_no_credit')._render(data),
+                                        'message_is_html': True,
```

Comment:
Meaning your previous commit does not work?

Reply:
It was working but showing escaped data instead of HTML note